models/Data.go

```diff
@@ -0,0 +1,5 @@
+package models
+
+// Data : Represent json model Data
```

Comment:
No need to use `:` inside comments, this will made the godoc ugly

Make complete comments (see how other Go packages (Mux, Cors, Websocket...) did)